pkg/config/builder_test.go

```diff
@@ -374,6 +378,7 @@ func TestLoadDir(t *testing.T) {
 	l.File(dir)
 
 	cfg, err := l.Build()
+	defer cfg.ShutDown()
```

Comment:
I think that this should go after checking `err`, as if `err` is not `nil` then the configuration shouldn't have been created, and `ShutDown()` should have no effect.

Reply:
:+1: